include/openssl/rsa.h

```diff
@@ -545,6 +545,10 @@ OPENSSL_EXPORT int RSA_get_ex_new_index(long argl, void *argp,
 OPENSSL_EXPORT int RSA_set_ex_data(RSA *rsa, int idx, void *arg);
 OPENSSL_EXPORT void *RSA_get_ex_data(const RSA *rsa, int idx);
 
+// RSA_pkey_ctx_ctrl is a wrapper of EVP_PKEY_CTX_ctrl.
```

Comment:
Check out other function descriptions and how `||` are used around parameters, constants, etc.

Reply:
Sorry, I don't get this comment.

Reply:
It is somewhat standard in AWS-LC to surround parameter names and function names with `||` in function descriptions. Just one random example: https://github.com/awslabs/aws-lc/blob/main/include/openssl/aead.h#L242

Reply:
Done.